custom/panel_templates/Default/template.php

```diff
@@ -232,7 +232,7 @@ public function onPageLoad() {
                             (defined('CONFIG_PATH') ? CONFIG_PATH : '') . '/core/assets/js/fomantic.toast.min.js' => [],
                         ]);
 
-                        $this->addJSScript(Input::createTinyEditor($this->_language, 'InputMaintenanceMessage'));
```

Comment:
You can do `$admin=true` instead of `null, false, true` so you don't have to set the earlier optional parameters

Reply:
Nope 😅
In php 8.0 you can use the named param feature but if we did $admin=true it would set the variable $admin to true in scope and then set whatever param is at that position to true

Reply:
huh.. too used to python I guess. thanks for the explanation